Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Repository/Query/HypergraphChildQuery.php

```diff
@@ -124,7 +124,7 @@ public function withOnlyTethered(): self
             AND cn.classification = :classification';
 
         $parameters = $this->parameters;
-        $parameters['classification'] = NodeAggregateClassification::CLASSIFICATION_TETHERED;
+        $parameters['classification'] = NodeAggregateClassification::CLASSIFICATION_TETHERED->value;
```

Comment:
Is that change related to this feature?

Reply:
nope, but it helps running the tests as it fixes something I seem to have missed